zeppelin-server/src/main/java/org/apache/zeppelin/socket/NotebookServer.java

```diff
@@ -251,27 +250,27 @@ public void onOpen(NotebookSocket conn) {
   @Override
   public void onMessage(NotebookSocket conn, String msg) {
     try {
-      Message messagereceived = deserializeMessage(msg);
-      if (messagereceived.op != OP.PING) {
-        LOG.debug("RECEIVE: " + messagereceived.op +
-            ", RECEIVE PRINCIPAL: " + messagereceived.principal +
-            ", RECEIVE TICKET: " + messagereceived.ticket +
-            ", RECEIVE ROLES: " + messagereceived.roles +
-            ", RECEIVE DATA: " + messagereceived.data);
+      Message receivedMessage = deserializeMessage(msg);
+      if (receivedMessage.op != OP.PING) {
+        LOG.debug("RECEIVE: " + receivedMessage.op +
```

Comment:
Please avoid the superfluous string concatenation and use this [method instead](http://www.slf4j.org/apidocs/org/slf4j/Logger.html#debug(java.lang.String,java.lang.Object...)).

Reply:
This PR is delayed too much, so I will fix it with a separate PR later.